msteams-platform/graph-api/rsc/preapproval-instruction-docs.md

```diff
@@ -0,0 +1,165 @@
+You must distinguish what type of RSC permissions you are pre-approving in the cmdlet you create. There are two options when pre-approving an app:
+
+|Configuration| Description|
+|---|---|
+|`ResourceSpecificApplicationPermissionsAllowedForChats`|Use this option when creating a pre-approval for RSC permissions for a team. A listing of these permissions can be found here
+|`ResourceSpecificApplicationPermissionsAllowedForTeams`|Use this option when creating a pre-approval for RSC permissions for a team. A listing of these permissions can be found here
+
+
+## Create a preapproval without a sensitivity label
+
+When you create your cmdlet you should use the option appropriate to the permissions included in the app you are trying to pre-approve. You can also include both if the app you are pre-approving has both Chat RSC and Team RSC permissions. Examples below show multiple ways to build cmndlets to pre-approve apps with varying permissions without specifying a sensitivity label:
+
+### Preapprove an app with Team RSC permissions:
+
+```powershell
+New-MgBetaTeamAppPreApproval -TeamsAppId c626ce8b-6d15-4c07-bfb1-a5fd0bc3c20e -ResourceSpecificApplicationPermissionsAllowedForTeams @('ChannelMessage.Read.Group')
+```
+### Preapprove an app with Chat RSC permissions:
+
+```powershell
+New-MgBetaTeamAppPreApproval -TeamsAppId c626ce8b-6d15-4c07-bfb1-a5fd0bc3c20e  -ResourceSpecificApplicationPermissionsAllowedForChats @('OnlineMeeting.ReadBasic.Chat')
+```
+### Preapprove an app with both Team and Chat RSC permissions:
+
+```powershell
+New-MgBetaTeamAppPreApproval -TeamsAppId c626ce8b-6d15-4c07-bfb1-a5fd0bc3c20e -ResourceSpecificApplicationPermissionsAllowedForChats @('OnlineMeeting.ReadBasic.Chat') -ResourceSpecificApplicationPermissionsAllowedForTeams @(‘ChannelMessage.Read.Group’)
+```
+
+>[!WARNING]
+>Your pre-approval creation may fail if you associate the wrong permissions with the wrong permission type in your cmdlet. Chat RSC permissions end in .Chat and Team RSC permissions end in .Group
+
+## Create a preapproval with a sensitivity label
+
+The same process can be used to create a pre-approval that applies to data within certain sensitivity labels in Microsoft Teams. You can use the `TeamLevelSensitivityLabelCondition` and `SpecificSensitivityLabel` arguments to define specific sensitivity labels to apply the RSC pre-approval policy to. Examples of cmndlets using these arguments are below:
+
+### Preapprove an app with Team RSC permissions and a defined sensitivity label:
+```powershell
+New-MgBetaTeamAppPreApproval -TeamsAppId c626ce8b-6d15-4c07-bfb1-a5fd0bc3c20e -ResourceSpecificApplicationPermissionsAllowedForTeams @('ChannelMessage.Read.Group') -TeamLevelSensitivityLabelCondition SpecificSensivityLabel -SpecificSensitivityLabelIdsApplicableToTeams @('4de11089-adb9-4be8-9b7a-8336be68f3c4')
+```
+### Preapprove an app with Chat RSC permissions and a defined sensitivity label:
```

Comment:
You should note that for now, a preapproval policy can only be assigned to one application.

Reply:
This document doesnt go into the internals of the policy creation. This will be confusing to the admin since this set of APIs dont expose any way to map policies to multiple apps.

Reply:
left without change after Subhajit's comment